packages/osc-ecommerce/app/components/CookieBanner/CookieBanner.scss

```
@@ -0,0 +1,27 @@
+/* stylelint-disable plugin/selector-bem-pattern */
+@import "../../../../osc-ui/src/styles/settings";
+@import "../../../../osc-ui/src/styles/tools";
+
+@layer components {
+    .c-cookie-banner {
+        position: fixed;
+        bottom: 1%;
+        z-index: 10;
```

recommend utilising the z-index map to control the layering
```suggestion
        @include z-index(popout);
```

---

Done